multisig/README.md

```diff
@@ -19,18 +19,28 @@ All operations going forward will require `K` signatures to be performed.
 
 There are number of different request types that multisig can confirm and execute:
 ```rust
- Transfer { receiver_id: AccountId, amount: U128 }
- AddKey { public_key: Base58PublicKey },
- DeleteKey { public_key: Base58PublicKey },
- FunctionCall {
-        contract_id: AccountId,
+enum MultisigRequestAction {
+  Transfer { amount: U128 },
+  AddKey { public_key: Base58PublicKey },
+  DeleteKey { public_key: Base58PublicKey },
+  FunctionCall {
         method_name: String,
         args: Base64VecU8,
         deposit: U128,
         gas: Gas
-    },
- SetNumConfirmations { num_confirmations: u32 }
-``` 
+  },
+  SetNumConfirmations { num_confirmations: u32 },
+  CreateAccount,
```

Comment:
let's also add DeployContract from that PR

Reply:
Let's also port the comments and other stuff from here: https://github.com/near/core-contracts/pull/66/files#diff-6620ce7e2df548737fd0e98c913fcc8aR22